template.tpl

```diff
@@ -781,8 +783,7 @@ scenarios:
     assertThat(JSON.parse(httpBody).data[0].user_data.st).isEqualTo(hashFunction('ca'));
     assertThat(JSON.parse(httpBody).data[0].user_data.zp).isEqualTo(hashFunction('94025'));
     assertThat(JSON.parse(httpBody).data[0].user_data.country).isEqualTo(hashFunction('usa'));
-
-- name: Set Meta cookies (fbp / fbc) if "extendCookies" checkbox is ticked
+- name: Set Meta cookies (fbp / fbc) if extendCookies checkbox is ticked
```

Comment:
These double quotes were removed since it will cause Google Server Side Container unable to run the tests since it recognized double quotes as invalid characters

<img width="1790" alt="Screen Shot 2022-05-07 at 1 47 12 AM" src="https://user-images.githubusercontent.com/7748552/167187413-104a4428-c8f1-4823-a7ad-b6911bd55491.png">